packages/cms-editor/src/utils/cms-utils.ts

```diff
@@ -1,7 +1,21 @@
-import type { MapStringString } from '@axonivy/cms-editor-protocol';
+import type {
+  CmsDataObject,
+  CmsDataObjectValues,
+  CmsFileDataObject,
+  CmsFolderDataObject,
+  CmsStringDataObject
+} from '@axonivy/cms-editor-protocol';
 
-export const removeValue = (values: MapStringString, languageTag: string) => {
+export const removeValue = <T extends CmsDataObjectValues>(values: T, languageTag: string) => {
   // eslint-disable-next-line @typescript-eslint/no-unused-vars
   const { [languageTag]: _, ...newValues } = values;
-  return newValues;
+  return newValues as T;
```

Comment:
looks wrong to me. Or maybe I don't fully understand this type.
But if values is T how can something inside be also T? isn't this CmsDataObjectValues all the language values of one content object?